test/system/tests/create.sh

```diff
@@ -49,27 +49,6 @@ cleanup() {
 }
 trap cleanup EXIT
 
-function yaml_get() {
```

Comment:
Same here, I think it will be easier to read if you first factor out yaml_get(), then on a subsequent patch add yaml_get_kv() and finally on another patch change the users.